From 32c18471bdc1bf757f327bdb98253e36aec54cc5 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 17:28:05 -0400
Subject: [redhat] configs: Enable NFSoRDMA server

Message-id: <cb113c6d379170d0e79aa12629b2bb32865eb36f.1411578494.git.dledford@redhat.com>
Patchwork-id: 96661
O-Subject: [Patch RHEL7 12/26] Enable NFSoRDMA server
Bugzilla: 1107621
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

We disabled this right before rhel7.0 went out due to a hard to track
down bug and lack of confidence in the overall code.  Since then, a
considerable amount of work has happened upstream, and been backported
to rhel7.1.  I think we can enable this now, and if we have any doubts,
then mark it as tech preview.

Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_SUNRPC_XPRT_RDMA_SERVER b/redhat/configs/generic/CONFIG_SUNRPC_XPRT_RDMA_SERVER
index 9c48290..d29c8b8 100644
--- a/redhat/configs/generic/CONFIG_SUNRPC_XPRT_RDMA_SERVER
+++ b/redhat/configs/generic/CONFIG_SUNRPC_XPRT_RDMA_SERVER
@@ -1 +1 @@
-# CONFIG_SUNRPC_XPRT_RDMA_SERVER is not set
+CONFIG_SUNRPC_XPRT_RDMA_SERVER=m
-- 
1.7.1